src/backend/base/langflow/components/embeddings/AstraVectorize.py

```diff
@@ -50,37 +50,38 @@ class AstraVectorize(Component):
     inputs = [
         DropdownInput(
             name="provider",
-            display_name="Provider name",
+            display_name="Provider",
             options=VECTORIZE_PROVIDERS_MAPPING.keys(),
             value="",
+            required=True,
         ),
         MessageTextInput(
             name="model_name",
-            display_name="Model name",
+            display_name="Model Name",
             info=f"The embedding model to use for the selected provider. Each provider has a different set of models "
-            f"available (full list at https://docs.datastax.com/en/astra-db-serverless/databases/embedding-generation.html):\n\n{VECTORIZE_MODELS_STR}",
+            f"available (https://docs.datastax.com/en/astra-db-serverless/databases/embedding-generation.html):\n\n{VECTORIZE_MODELS_STR}",
             required=True,
         ),
+        SecretStrInput(
+            name="provider_api_key",
+            display_name="Provider API Key",
+            info="An alternative to the Astra Authentication that passes an API key for the provider with each request to Astra DB. This may be used when Vectorize is configured for the collection, but no corresponding provider secret is stored within Astra's key management system.",
+        ),
```

Comment:
I made this appear by default. The reasoning is that 
a) This is the quickest path to testing out vectorize on a new collection
b) If the collection is already configured for vectorize, but the key was not stored in astra, this is (by my guess) the most common pattern for authentication.